README.md

```diff
@@ -31,13 +31,10 @@ are used as a relatively simple resource management tool to
 provide some fairness between service consumers when contending
 for limited service resources.
 
-To learn more...
+Please see the main Istio [README](https://raw.githubusercontent.com/istio/istio/master/README.md)
+file to learn about the overall Istio project and how to get in touch with us. To learn how you can
+contribute to any of the Istio components, including the mixer, please 
+see the Istio [contribution guidelines](https://raw.githubusercontent.com/istio/istio/master/CONTRIBUTING.md).
```

Comment:
These `raw.githubusercontent.com` links all go to unrendered markdown files. We should at least link to the rendered pages (`https://github.com/istio/istio/blob/master/CONTRIBUTING.md` in this case), or we could even create a github pages repo that we can link to (probably overkill given istio.io will serve that purpose eventually).